exercises/practice/alphametics/.docs/instructions.md

```diff
@@ -28,5 +28,4 @@ Each letter must represent a different digit, and the leading digit of a multi-d
 
 Write a function to solve alphametics puzzles.
 
-You may (or may not!) want to call the function `permutations(a, t)` from [Combinatorics.jl](https://github.com/JuliaMath/Combinatorics.jl) in your solution.
-If you would like to use it, you can include the **read-only** file `permutations.jl` with `include("permutations.jl")`.
```

Comment:
Julia-specific changes like this should generally be retained.

Reply:
Yeah, those should be in an append for ease of maintenance.